src/api/python/_api.py

```
@@ -380,6 +394,13 @@ def get_backend_params(self, backend: str) -> dict[str, str]:
     """
 
     def create_backend(self, backend: str, initParams: dict[str, str] = {}):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `create_backend`

Add return type annotation: `None`

(ANN202)